introduction_to_amazon_algorithms/jumpstart_image_classification/Amazon_JumpStart_Image_Classification.ipynb

```diff
@@ -2,15 +2,15 @@
  "cells": [
```

Comment:
<p>Be consistent throughout the notebook in use of <code>fine-tuned</code> versus <code>finetuned</code>   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3429/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Addressed.</p>